lib/lora/API.md

```diff
@@ -0,0 +1,315 @@
+# Heltec LoRa basic library API
```

Comment:
Can you add the library via PlatformIO's library manager? https://platformio.org/lib/show/6051/Heltec%20ESP32%20Dev-Boards

Is there a library that is not specific to the Heltec board? maybe https://platformio.org/lib/show/1167/LoRa

Reply:
Can do. The Heltec library should be generic for most of the SX chipsets, but lacks LoRaWAN support.  It was a "nice low bar" to get started.